src/content/config.ts

```diff
@@ -0,0 +1,225 @@
+import { z, defineCollection } from "astro:content"
+
+const anyApiCollection = defineCollection({
+  type: "content",
+  schema: z
+    .object({
+      section: z.string(),
```

Comment:
There is only a limited amount of sections that can be used throughout the app, so maybe we could narrow down this type to a [zod enum](https://github.com/colinhacks/zod#zod-enums) for extra safety